st2client/tests/base.py

```diff
@@ -86,6 +86,8 @@ def setUp(self):
             if var in os.environ:
                 del os.environ[var]
 
+        os.environ['ST2_SKIP_CLI_CONFIG'] = '1'
```

Comment:
Heh, that's actually kinda related to the auth change we talked about.

In short - there are scenarios where you want don't want to use the CLI config (e.g. you only want to use values provided as the command line arguments).

Without this option, you need to "temporary" remove or move the config.